2023-01-29 20:53:44,734, ERROR, The following variable(s) are missing: ['TELEGRAM_CHAT_ID'], Строка № 48
2023-01-29 20:55:58,208, ERROR, Отсутствует обязательная переменная окружения:['TELEGRAM_CHAT_ID'] Программа принудительно остановлена., Строка № 48
2023-01-29 20:56:34,143, ERROR, Отсутствует обязательная переменная окружения:['TELEGRAM_CHAT_ID']. Программа принудительно остановлена, Строка № 48
2023-01-29 20:59:32,582, INFO, запущена функция, Строка № 43
2023-01-29 20:59:32,602, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Строка № 1003
2023-01-29 20:59:36,110, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Строка № 456
2023-01-29 20:59:41,160, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Строка № 1003
2023-01-29 20:59:42,699, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 500 None, Строка № 456
2023-01-29 21:32:10,442, INFO, Запущена функция, Строка № 39
2023-01-29 21:32:10,551, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Строка № 1003
2023-01-29 21:32:12,262, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Строка № 456
2023-01-29 21:32:18,632, DEBUG, Entering: send_message, Строка № 129
2023-01-29 21:32:19,996, DEBUG, {'text': 'Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!', 'delete_chat_photo': False, 'supergroup_chat_created': False, 'photo': [], 'new_chat_members': [], 'chat': {'username': 'slexvik', 'id': 397784581, 'first_name': 'Alexey', 'last_name': 'S', 'type': 'private'}, 'date': 1675002739, 'new_chat_photo': [], 'entities': [], 'group_chat_created': False, 'caption_entities': [], 'message_id': 127, 'channel_chat_created': False, 'from': {'id': 5762476788, 'is_bot': True, 'first_name': 'My trainer robot', 'username': 'MyTrainerRobot'}}, Строка № 131
2023-01-29 21:32:20,005, DEBUG, Exiting: send_message, Строка № 132
2023-01-29 21:32:56,832, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Строка № 1003
2023-01-29 21:32:58,498, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Строка № 456
2023-01-29 21:54:31,227, INFO, Запущена функция, Строка № 39
2023-01-29 21:55:49,207, INFO, Запущена функция, Строка № 39
2023-01-29 21:55:49,237, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Строка № 1003
2023-01-29 21:55:51,204, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Строка № 456
2023-01-29 21:55:51,230, DEBUG, Entering: send_message, Строка № 129
2023-01-29 21:55:54,210, DEBUG, {'new_chat_members': [], 'photo': [], 'message_id': 128, 'supergroup_chat_created': False, 'channel_chat_created': False, 'delete_chat_photo': False, 'group_chat_created': False, 'new_chat_photo': [], 'date': 1675004153, 'text': 'Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!', 'chat': {'first_name': 'Alexey', 'id': 397784581, 'last_name': 'S', 'username': 'slexvik', 'type': 'private'}, 'entities': [], 'caption_entities': [], 'from': {'username': 'MyTrainerRobot', 'id': 5762476788, 'is_bot': True, 'first_name': 'My trainer robot'}}, Строка № 131
2023-01-29 21:55:54,222, DEBUG, Exiting: send_message, Строка № 132
2023-01-29 21:55:59,247, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Строка № 1003
2023-01-29 21:56:00,903, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Строка № 456
2023-01-29 21:56:05,962, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Строка № 1003
2023-01-29 21:56:07,546, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Строка № 456
2023-01-29 21:56:12,603, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Строка № 1003
2023-01-29 21:56:15,458, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Строка № 456
2023-01-29 21:56:20,497, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Строка № 1003
2023-01-29 21:56:26,220, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Строка № 456
2023-01-29 21:56:31,269, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Строка № 1003
2023-01-29 21:56:32,828, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Строка № 456
2023-01-29 21:58:45,335, INFO, Запущена функция, Строка № 39
2023-01-29 21:58:45,352, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Строка № 1003
2023-01-29 21:58:49,828, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Строка № 456
2023-01-29 21:58:49,849, DEBUG, Entering: send_message, Строка № 129
2023-01-29 21:58:51,782, DEBUG, {'new_chat_photo': [], 'chat': {'type': 'private', 'last_name': 'S', 'id': 397784581, 'username': 'slexvik', 'first_name': 'Alexey'}, 'text': 'Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!', 'caption_entities': [], 'channel_chat_created': False, 'entities': [], 'date': 1675004330, 'photo': [], 'supergroup_chat_created': False, 'new_chat_members': [], 'delete_chat_photo': False, 'group_chat_created': False, 'message_id': 129, 'from': {'id': 5762476788, 'username': 'MyTrainerRobot', 'is_bot': True, 'first_name': 'My trainer robot'}}, Строка № 131
2023-01-29 21:58:51,790, DEBUG, Exiting: send_message, Строка № 132
2023-01-29 21:58:51,790, INFO, Сообщение отправлено Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!, Строка № 113
2023-01-29 21:58:56,802, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Строка № 1003
2023-01-29 21:58:58,447, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Строка № 456
2023-01-29 21:59:03,481, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Строка № 1003
2023-01-29 21:59:06,872, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Строка № 456
2023-01-29 21:59:16,165, INFO, Запущена функция, Строка № 39
2023-01-29 21:59:16,181, ERROR, Сбой в работе программы: No connection adapters were found for 'htps://practicum.yandex.ru/api/user_api/homework_statuses/', Строка № 122
2023-01-29 22:01:20,086, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 39
2023-01-29 22:01:20,103, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:01:22,261, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:01:22,275, DEBUG, Entering: send_message, Файл - bot.py, Функция - decorator,  Строка № 129
2023-01-29 22:01:23,562, DEBUG, {'channel_chat_created': False, 'photo': [], 'new_chat_photo': [], 'group_chat_created': False, 'entities': [], 'text': 'Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!', 'supergroup_chat_created': False, 'caption_entities': [], 'message_id': 130, 'date': 1675004482, 'new_chat_members': [], 'chat': {'type': 'private', 'last_name': 'S', 'username': 'slexvik', 'first_name': 'Alexey', 'id': 397784581}, 'delete_chat_photo': False, 'from': {'first_name': 'My trainer robot', 'is_bot': True, 'username': 'MyTrainerRobot', 'id': 5762476788}}, Файл - bot.py, Функция - decorator,  Строка № 131
2023-01-29 22:01:23,565, DEBUG, Exiting: send_message, Файл - bot.py, Функция - decorator,  Строка № 132
2023-01-29 22:01:23,565, INFO, Сообщение отправлено Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!, Файл - homework.py, Функция - main,  Строка № 113
2023-01-29 22:01:28,574, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:01:30,149, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:01:35,215, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:01:37,381, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:01:42,439, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:01:44,095, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:01:49,144, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:01:50,759, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:03:12,889, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 39
2023-01-29 22:03:12,902, ERROR, Сбой в работе программы: No connection adapters were found for 'htps://practicum.yandex.ru/api/user_api/homework_statuses/', Файл - homework.py, Функция - main,  Строка № 122
2023-01-29 22:05:34,284, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 39
2023-01-29 22:05:34,299, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:05:36,964, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:05:36,983, DEBUG, Entering: send_message, Файл - bot.py, Функция - decorator,  Строка № 129
2023-01-29 22:05:39,168, DEBUG, {'text': 'Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!', 'message_id': 131, 'delete_chat_photo': False, 'photo': [], 'chat': {'last_name': 'S', 'id': 397784581, 'type': 'private', 'first_name': 'Alexey', 'username': 'slexvik'}, 'date': 1675004737, 'entities': [], 'new_chat_members': [], 'caption_entities': [], 'supergroup_chat_created': False, 'channel_chat_created': False, 'group_chat_created': False, 'new_chat_photo': [], 'from': {'username': 'MyTrainerRobot', 'id': 5762476788, 'first_name': 'My trainer robot', 'is_bot': True}}, Файл - bot.py, Функция - decorator,  Строка № 131
2023-01-29 22:05:39,182, DEBUG, Exiting: send_message, Файл - bot.py, Функция - decorator,  Строка № 132
2023-01-29 22:05:39,187, INFO, Сообщение отправлено Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!, Файл - homework.py, Функция - main,  Строка № 113
2023-01-29 22:05:39,193, DEBUG, Статус ДЗ не изменился, Файл - homework.py, Функция - main,  Строка № 116
2023-01-29 22:05:44,206, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:05:45,799, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:05:45,825, DEBUG, Статус ДЗ не изменился, Файл - homework.py, Функция - main,  Строка № 116
2023-01-29 22:05:50,856, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:05:53,774, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:05:53,791, DEBUG, Статус ДЗ не изменился, Файл - homework.py, Функция - main,  Строка № 116
2023-01-29 22:05:58,822, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:06:02,033, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:06:02,055, DEBUG, Статус ДЗ не изменился, Файл - homework.py, Функция - main,  Строка № 116
2023-01-29 22:06:07,080, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:06:08,973, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:06:08,981, DEBUG, Статус ДЗ не изменился, Файл - homework.py, Функция - main,  Строка № 116
2023-01-29 22:06:39,289, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 39
2023-01-29 22:06:39,304, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:06:42,997, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:06:43,016, INFO, Сообщение отправлено Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!, Файл - homework.py, Функция - main,  Строка № 113
2023-01-29 22:06:48,043, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:06:52,124, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:06:52,156, DEBUG, Статус ДЗ не изменился, Файл - homework.py, Функция - main,  Строка № 117
2023-01-29 22:06:57,180, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:06:58,807, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:06:58,828, DEBUG, Статус ДЗ не изменился, Файл - homework.py, Функция - main,  Строка № 117
2023-01-29 22:12:04,243, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 39
2023-01-29 22:12:04,251, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:12:06,824, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:12:06,834, ERROR, Сбой в работе программы: exceptions must derive from BaseException, Файл - homework.py, Функция - main,  Строка № 125
2023-01-29 22:12:25,961, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 39
2023-01-29 22:12:25,991, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:12:27,732, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:12:27,776, INFO, Сообщение отправлено Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!, Файл - homework.py, Функция - main,  Строка № 115
2023-01-29 22:14:22,178, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 38
2023-01-29 22:14:22,203, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-29 22:14:24,111, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-29 22:14:24,150, ERROR, Неожиданный статус домашней работы 'approved', Файл - homework.py, Функция - parse_status,  Строка № 91
2023-01-29 22:14:24,152, ERROR, Сбой в работе программы: 'approved', Файл - homework.py, Функция - main,  Строка № 125
2023-01-30 11:49:56,181, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 38
2023-01-30 11:49:56,208, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 12:18:37,202, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 38
2023-01-30 12:19:10,799, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 12:19:12,286, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675055944 HTTP/1.1" 200 42, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-30 12:20:08,542, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 12:20:09,954, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675055997 HTTP/1.1" 200 42, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-30 13:09:07,894, INFO, Начало проверки, Файл - homework.py, Функция - main,  Строка № 100
2023-01-30 13:09:16,113, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 38
2023-01-30 13:09:48,887, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 13:09:50,792, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-30 13:11:38,504, DEBUG, Entering: send_message, Файл - bot.py, Функция - decorator,  Строка № 129
2023-01-30 13:11:39,598, DEBUG, {'new_chat_members': [], 'chat': {'last_name': 'S', 'id': 397784581, 'type': 'private', 'first_name': 'Alexey', 'username': 'slexvik'}, 'caption_entities': [], 'supergroup_chat_created': False, 'message_id': 132, 'group_chat_created': False, 'date': 1675059097, 'delete_chat_photo': False, 'text': 'Сбой в работе программы: (AttributeError("\'dict\' object has no attribute \'status_code\'"), \'Не верный код ответа параметры запроса: url = {url},\')', 'channel_chat_created': False, 'photo': [], 'entities': [], 'new_chat_photo': [], 'from': {'first_name': 'My trainer robot', 'username': 'MyTrainerRobot', 'is_bot': True, 'id': 5762476788}}, Файл - bot.py, Функция - decorator,  Строка № 131
2023-01-30 13:11:39,598, DEBUG, Exiting: send_message, Файл - bot.py, Функция - decorator,  Строка № 132
2023-01-30 13:11:41,615, DEBUG, Сообщение отправлено: Сбой в работе программы: (AttributeError("'dict' object has no attribute 'status_code'"), 'Не верный код ответа параметры запроса: url = {url},'), Файл - homework.py, Функция - send_message,  Строка № 58
2023-01-30 13:12:52,666, INFO, Начало проверки, Файл - homework.py, Функция - main,  Строка № 99
2023-01-30 13:12:57,259, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 38
2023-01-30 13:13:35,168, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 13:13:37,120, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-30 13:14:41,309, DEBUG, Entering: send_message, Файл - bot.py, Функция - decorator,  Строка № 129
2023-01-30 13:14:42,385, DEBUG, {'entities': [], 'new_chat_photo': [], 'group_chat_created': False, 'message_id': 133, 'chat': {'type': 'private', 'last_name': 'S', 'first_name': 'Alexey', 'id': 397784581, 'username': 'slexvik'}, 'new_chat_members': [], 'caption_entities': [], 'channel_chat_created': False, 'text': "Сбой в работе программы: 'dict' object has no attribute 'status_code'", 'photo': [], 'supergroup_chat_created': False, 'delete_chat_photo': False, 'date': 1675059280, 'from': {'is_bot': True, 'first_name': 'My trainer robot', 'id': 5762476788, 'username': 'MyTrainerRobot'}}, Файл - bot.py, Функция - decorator,  Строка № 131
2023-01-30 13:14:42,401, DEBUG, Exiting: send_message, Файл - bot.py, Функция - decorator,  Строка № 132
2023-01-30 13:14:44,155, DEBUG, Сообщение отправлено: Сбой в работе программы: 'dict' object has no attribute 'status_code', Файл - homework.py, Функция - send_message,  Строка № 58
2023-01-30 13:14:58,322, ERROR, Сбой в работе программы: 'dict' object has no attribute 'status_code', Файл - homework.py, Функция - main,  Строка № 122
2023-01-30 13:16:45,864, INFO, Начало проверки, Файл - homework.py, Функция - main,  Строка № 99
2023-01-30 13:16:45,864, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 38
2023-01-30 13:16:55,010, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 13:16:56,594, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-30 13:17:40,165, DEBUG, Entering: send_message, Файл - bot.py, Функция - decorator,  Строка № 129
2023-01-30 13:17:41,329, DEBUG, {'chat': {'username': 'slexvik', 'last_name': 'S', 'type': 'private', 'id': 397784581, 'first_name': 'Alexey'}, 'entities': [], 'delete_chat_photo': False, 'new_chat_photo': [], 'photo': [], 'text': 'Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!', 'group_chat_created': False, 'caption_entities': [], 'date': 1675059459, 'message_id': 134, 'new_chat_members': [], 'channel_chat_created': False, 'supergroup_chat_created': False, 'from': {'is_bot': True, 'username': 'MyTrainerRobot', 'id': 5762476788, 'first_name': 'My trainer robot'}}, Файл - bot.py, Функция - decorator,  Строка № 131
2023-01-30 13:17:41,329, DEBUG, Exiting: send_message, Файл - bot.py, Функция - decorator,  Строка № 132
2023-01-30 13:17:50,114, DEBUG, Сообщение отправлено: Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!, Файл - homework.py, Функция - send_message,  Строка № 58
2023-01-30 13:52:34,470, INFO, Начало проверки, Файл - homework.py, Функция - main,  Строка № 102
2023-01-30 13:52:34,473, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 36
2023-01-30 13:52:34,572, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 13:52:36,881, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-30 13:53:36,443, INFO, Начало проверки, Файл - homework.py, Функция - main,  Строка № 102
2023-01-30 13:54:37,518, INFO, Запущена функция, Файл - homework.py, Функция - check_tokens,  Строка № 36
2023-01-30 13:55:51,438, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 13:55:52,930, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-30 13:58:23,432, DEBUG, Entering: send_message, Файл - bot.py, Функция - decorator,  Строка № 129
2023-01-30 13:58:24,619, DEBUG, {'channel_chat_created': False, 'new_chat_photo': [], 'chat': {'username': 'slexvik', 'id': 397784581, 'first_name': 'Alexey', 'last_name': 'S', 'type': 'private'}, 'new_chat_members': [], 'entities': [], 'caption_entities': [], 'delete_chat_photo': False, 'photo': [], 'date': 1675061902, 'group_chat_created': False, 'supergroup_chat_created': False, 'message_id': 135, 'text': "Сбой в работе программы: 'approved'", 'from': {'username': 'MyTrainerRobot', 'id': 5762476788, 'is_bot': True, 'first_name': 'My trainer robot'}}, Файл - bot.py, Функция - decorator,  Строка № 131
2023-01-30 13:58:24,630, DEBUG, Exiting: send_message, Файл - bot.py, Функция - decorator,  Строка № 132
2023-01-30 16:17:40,174, INFO, Начало проверки, Файл - homework.py, Функция - main,  Строка № 107
2023-01-30 16:17:44,678, INFO, Старт проверки наличия токенов, Файл - homework.py, Функция - check_tokens,  Строка № 38
2023-01-30 16:18:16,940, INFO, Старт запуская обращения к АПИ, Файл - homework.py, Функция - get_api_answer,  Строка № 63
2023-01-30 16:18:21,075, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 16:18:23,087, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-30 16:18:36,526, INFO, Старт проверки АПИ на соответсвие документации, Файл - homework.py, Функция - check_response,  Строка № 79
2023-01-30 16:19:12,357, INFO, Старт проверки статусов ДЗ, Файл - homework.py, Функция - parse_status,  Строка № 90
2023-01-30 16:21:15,403, INFO, Начало проверки, Файл - homework.py, Функция - main,  Строка № 107
2023-01-30 16:21:15,407, INFO, Старт проверки наличия токенов, Файл - homework.py, Функция - check_tokens,  Строка № 38
2023-01-30 16:21:15,413, INFO, Старт запуская обращения к АПИ, Файл - homework.py, Функция - get_api_answer,  Строка № 63
2023-01-30 16:21:15,521, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 16:21:17,023, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-30 16:21:17,069, INFO, Старт проверки АПИ на соответсвие документации, Файл - homework.py, Функция - check_response,  Строка № 79
2023-01-30 16:21:17,072, INFO, Старт проверки статусов ДЗ, Файл - homework.py, Функция - parse_status,  Строка № 90
2023-01-30 16:22:34,630, INFO, Начало проверки, Файл - homework.py, Функция - main,  Строка № 107
2023-01-30 16:22:34,630, INFO, Старт проверки наличия токенов, Файл - homework.py, Функция - check_tokens,  Строка № 38
2023-01-30 16:22:34,644, INFO, Старт запуская обращения к АПИ, Файл - homework.py, Функция - get_api_answer,  Строка № 63
2023-01-30 16:22:34,730, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 16:22:37,402, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-30 16:22:37,443, INFO, Старт проверки АПИ на соответсвие документации, Файл - homework.py, Функция - check_response,  Строка № 79
2023-01-30 16:22:37,443, INFO, Старт проверки статусов ДЗ, Файл - homework.py, Функция - parse_status,  Строка № 90
2023-01-30 16:24:34,378, INFO, Начало проверки, Файл - homework.py, Функция - main,  Строка № 106
2023-01-30 16:24:34,382, INFO, Старт проверки наличия токенов, Файл - homework.py, Функция - check_tokens,  Строка № 37
2023-01-30 16:24:34,389, INFO, Старт запуская обращения к АПИ, Файл - homework.py, Функция - get_api_answer,  Строка № 62
2023-01-30 16:24:34,477, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 16:24:36,911, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-30 16:24:36,943, INFO, Старт проверки АПИ на соответсвие документации, Файл - homework.py, Функция - check_response,  Строка № 78
2023-01-30 16:24:36,946, INFO, Старт проверки статусов ДЗ, Файл - homework.py, Функция - parse_status,  Строка № 89
2023-01-30 16:33:27,164, INFO, Начало проверки, Файл - homework.py, Функция - main,  Строка № 107
2023-01-30 16:33:27,164, INFO, Старт проверки наличия токенов, Файл - homework.py, Функция - check_tokens,  Строка № 37
2023-01-30 16:33:27,173, INFO, Старт запуская обращения к АПИ, Файл - homework.py, Функция - get_api_answer,  Строка № 62
2023-01-30 16:33:27,269, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 16:33:29,219, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-30 16:33:29,272, INFO, Старт проверки АПИ на соответсвие документации, Файл - homework.py, Функция - check_response,  Строка № 78
2023-01-30 16:33:29,276, INFO, Старт проверки статусов ДЗ, Файл - homework.py, Функция - parse_status,  Строка № 89
2023-01-30 16:34:03,614, INFO, Начало проверки, Файл - homework.py, Функция - main,  Строка № 107
2023-01-30 16:34:03,618, INFO, Старт проверки наличия токенов, Файл - homework.py, Функция - check_tokens,  Строка № 37
2023-01-30 16:34:03,624, INFO, Старт запуская обращения к АПИ, Файл - homework.py, Функция - get_api_answer,  Строка № 62
2023-01-30 16:34:03,730, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443, Файл - connectionpool.py, Функция - _new_conn,  Строка № 1003
2023-01-30 16:34:05,424, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372, Файл - connectionpool.py, Функция - _make_request,  Строка № 456
2023-01-30 16:34:05,464, INFO, Старт проверки АПИ на соответсвие документации, Файл - homework.py, Функция - check_response,  Строка № 78
2023-01-30 16:34:05,468, INFO, Старт проверки статусов ДЗ, Файл - homework.py, Функция - parse_status,  Строка № 89
2023-01-30 16:34:19,376, ERROR, Неожиданный статус домашней работы 'approved', Файл - homework.py, Функция - parse_status,  Строка № 101
2023-01-30 16:34:33,996, DEBUG, Entering: send_message, Файл - bot.py, Функция - decorator,  Строка № 129
2023-01-30 16:34:35,127, DEBUG, {'date': 1675071273, 'message_id': 136, 'delete_chat_photo': False, 'group_chat_created': False, 'text': "Сбой в работе программы: 'approved'", 'photo': [], 'new_chat_members': [], 'channel_chat_created': False, 'new_chat_photo': [], 'caption_entities': [], 'chat': {'type': 'private', 'id': 397784581, 'username': 'slexvik', 'last_name': 'S', 'first_name': 'Alexey'}, 'supergroup_chat_created': False, 'entities': [], 'from': {'is_bot': True, 'first_name': 'My trainer robot', 'username': 'MyTrainerRobot', 'id': 5762476788}}, Файл - bot.py, Функция - decorator,  Строка № 131
2023-01-30 16:34:35,133, DEBUG, Exiting: send_message, Файл - bot.py, Функция - decorator,  Строка № 132
2023-01-30 16:34:36,702, DEBUG, Сообщение отправлено: Сбой в работе программы: 'approved', Файл - homework.py, Функция - send_message,  Строка № 57
2023-01-30 16:34:43,223, ERROR, Сбой в работе программы: 'approved', Файл - homework.py, Функция - main,  Строка № 130
2023-01-31 11:57:07,311, INFO, Начало проверки,Файл - homework.py, Строка № 107
2023-01-31 11:57:07,311, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 39
2023-01-31 11:57:07,311, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 66
2023-01-31 11:57:07,345, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-01-31 11:57:08,902, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675141027 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-01-31 11:57:08,911, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 83
2023-01-31 11:59:15,149, INFO, Начало проверки,Файл - homework.py, Строка № 107
2023-01-31 11:59:15,149, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 39
2023-01-31 11:59:15,149, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 66
2023-01-31 11:59:15,184, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-01-31 11:59:16,762, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675141155 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-01-31 11:59:16,795, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 83
2023-01-31 11:59:51,598, INFO, Начало проверки,Файл - homework.py, Строка № 107
2023-01-31 11:59:51,598, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 39
2023-01-31 11:59:51,609, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 66
2023-01-31 11:59:51,689, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-01-31 11:59:53,910, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675141191 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-01-31 12:00:00,229, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 83
2023-01-31 12:01:10,828, INFO, Начало проверки,Файл - homework.py, Строка № 107
2023-01-31 12:01:10,828, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 39
2023-01-31 12:01:10,828, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 66
2023-01-31 12:01:10,856, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-01-31 12:01:12,561, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372,Файл - connectionpool.py, Строка № 456
2023-01-31 12:01:12,596, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 83
2023-01-31 12:01:12,598, INFO, Старт проверки статусов ДЗ,Файл - homework.py, Строка № 94
2023-01-31 12:01:12,600, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-01-31 12:01:13,700, DEBUG, {'new_chat_photo': [], 'entities': [], 'new_chat_members': [], 'photo': [], 'caption_entities': [], 'date': 1675141273, 'delete_chat_photo': False, 'chat': {'username': 'slexvik', 'last_name': 'S', 'first_name': 'Alexey', 'type': 'private', 'id': 397784581}, 'message_id': 137, 'text': 'Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!', 'supergroup_chat_created': False, 'group_chat_created': False, 'channel_chat_created': False, 'from': {'is_bot': True, 'username': 'MyTrainerRobot', 'id': 5762476788, 'first_name': 'My trainer robot'}},Файл - bot.py, Строка № 131
2023-01-31 12:01:13,700, DEBUG, Exiting: send_message,Файл - bot.py, Строка № 132
2023-01-31 12:01:13,716, DEBUG, Сообщение отправлено: Изменился статус проверки работы "Slexvik__hw05_final.zip". Работа проверена: ревьюеру всё понравилось. Ура!,Файл - homework.py, Строка № 61
2023-01-31 12:01:18,731, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 66
2023-01-31 12:01:18,767, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-01-31 12:01:20,146, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372,Файл - connectionpool.py, Строка № 456
2023-01-31 12:01:20,174, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 83
2023-01-31 12:01:20,176, DEBUG, Статус ДЗ не изменился,Файл - homework.py, Строка № 124
2023-01-31 12:01:25,188, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 66
2023-01-31 12:01:25,197, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-01-31 12:01:26,572, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372,Файл - connectionpool.py, Строка № 456
2023-01-31 12:01:26,580, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 83
2023-01-31 12:01:26,582, DEBUG, Статус ДЗ не изменился,Файл - homework.py, Строка № 124
2023-01-31 12:01:31,589, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 66
2023-01-31 12:01:31,609, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-01-31 12:01:32,967, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372,Файл - connectionpool.py, Строка № 456
2023-01-31 12:01:33,010, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 83
2023-01-31 12:01:33,013, DEBUG, Статус ДЗ не изменился,Файл - homework.py, Строка № 124
2023-01-31 12:01:38,025, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 66
2023-01-31 12:01:38,071, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-01-31 12:01:39,448, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1672558471 HTTP/1.1" 200 372,Файл - connectionpool.py, Строка № 456
2023-01-31 12:01:39,481, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 83
2023-01-31 12:01:39,482, DEBUG, Статус ДЗ не изменился,Файл - homework.py, Строка № 124
2023-01-31 12:30:57,437, INFO, Начало проверки,Файл - homework.py, Строка № 107
2023-01-31 12:30:57,437, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 39
2023-01-31 12:30:57,453, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 66
2023-01-31 12:31:08,054, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-01-31 12:31:09,331, DEBUG, {'photo': [], 'channel_chat_created': False, 'entities': [], 'delete_chat_photo': False, 'new_chat_photo': [], 'supergroup_chat_created': False, 'chat': {'type': 'private', 'username': 'slexvik', 'last_name': 'S', 'id': 397784581, 'first_name': 'Alexey'}, 'text': "Сбой в работе программы: No connection adapters were found for 'htps://practicum.yandex.ru/api/user_api/homework_statuses/'", 'new_chat_members': [], 'group_chat_created': False, 'caption_entities': [], 'date': 1675143069, 'message_id': 138, 'from': {'first_name': 'My trainer robot', 'is_bot': True, 'id': 5762476788, 'username': 'MyTrainerRobot'}},Файл - bot.py, Строка № 131
2023-01-31 12:31:09,340, DEBUG, Exiting: send_message,Файл - bot.py, Строка № 132
2023-01-31 12:31:18,101, DEBUG, Сообщение отправлено: Сбой в работе программы: No connection adapters were found for 'htps://practicum.yandex.ru/api/user_api/homework_statuses/',Файл - homework.py, Строка № 61
2023-01-31 12:31:26,112, ERROR, Сбой в работе программы: No connection adapters were found for 'htps://practicum.yandex.ru/api/user_api/homework_statuses/',Файл - homework.py, Строка № 131
2023-01-31 12:31:38,797, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 66
2023-01-31 12:47:05,268, INFO, Начало проверки,Файл - homework.py, Строка № 107
2023-01-31 12:47:05,284, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 39
2023-01-31 12:47:05,290, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 66
2023-01-31 12:47:05,385, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-01-31 12:47:06,958, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675144025 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-01-31 12:47:10,069, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 83
2023-01-31 13:00:45,416, INFO, Начало проверки,Файл - homework.py, Строка № 107
2023-01-31 13:00:45,420, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 39
2023-01-31 13:00:45,426, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 66
2023-01-31 13:00:45,565, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-01-31 13:00:47,055, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675144845 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-02-01 13:15:11,327, INFO, Начало проверки,Файл - homework.py, Строка № 108
2023-02-01 13:15:11,327, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 39
2023-02-01 13:15:11,339, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 67
2023-02-01 13:15:11,432, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-01 13:15:13,597, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-01 13:15:16,678, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 84
2023-02-01 13:15:33,509, INFO, Старт проверки статусов ДЗ,Файл - homework.py, Строка № 95
2023-02-01 13:15:43,449, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-02-01 13:15:45,565, DEBUG, {'text': 'Изменился статус проверки работы "Slexvik__homework_bot.zip". Работа проверена: у ревьюера есть замечания.', 'delete_chat_photo': False, 'photo': [], 'group_chat_created': False, 'supergroup_chat_created': False, 'new_chat_members': [], 'chat': {'last_name': 'S', 'first_name': 'Alexey', 'id': 397784581, 'type': 'private', 'username': 'slexvik'}, 'new_chat_photo': [], 'caption_entities': [], 'date': 1675232144, 'channel_chat_created': False, 'message_id': 139, 'entities': [], 'from': {'first_name': 'My trainer robot', 'username': 'MyTrainerRobot', 'is_bot': True, 'id': 5762476788}},Файл - bot.py, Строка № 131
2023-02-01 13:15:45,573, DEBUG, Exiting: send_message,Файл - bot.py, Строка № 132
2023-02-01 13:15:56,412, DEBUG, Сообщение отправлено: Изменился статус проверки работы "Slexvik__homework_bot.zip". Работа проверена: у ревьюера есть замечания.,Файл - homework.py, Строка № 62
2023-02-01 13:18:33,242, INFO, Начало проверки,Файл - homework.py, Строка № 108
2023-02-01 13:18:33,242, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 39
2023-02-01 13:18:33,260, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 67
2023-02-01 13:18:33,350, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-01 13:18:35,429, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675232313 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-02-01 13:18:39,828, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 84
2023-02-01 17:34:05,204, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 32
2023-02-01 17:34:05,204, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 59
2023-02-01 17:34:05,234, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-01 17:34:06,862, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-01 17:34:06,896, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-01 17:34:06,898, INFO, Старт проверки статусов ДЗ,Файл - homework.py, Строка № 85
2023-02-01 17:34:06,901, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-02-01 17:34:08,099, DEBUG, {'photo': [], 'caption_entities': [], 'date': 1675247646, 'text': 'Изменился статус проверки работы "Slexvik__homework_bot.zip". Работа проверена: у ревьюера есть замечания.', 'entities': [], 'message_id': 141, 'delete_chat_photo': False, 'new_chat_members': [], 'supergroup_chat_created': False, 'group_chat_created': False, 'channel_chat_created': False, 'chat': {'id': 397784581, 'username': 'slexvik', 'last_name': 'S', 'first_name': 'Alexey', 'type': 'private'}, 'new_chat_photo': [], 'from': {'id': 5762476788, 'is_bot': True, 'first_name': 'My trainer robot', 'username': 'MyTrainerRobot'}},Файл - bot.py, Строка № 131
2023-02-01 17:34:08,099, DEBUG, Exiting: send_message,Файл - bot.py, Строка № 132
2023-02-01 17:34:08,099, DEBUG, Сообщение отправлено: Изменился статус проверки работы "Slexvik__homework_bot.zip". Работа проверена: у ревьюера есть замечания.,Файл - homework.py, Строка № 54
2023-02-01 17:34:37,176, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 32
2023-02-01 17:34:37,194, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 59
2023-02-01 17:34:37,280, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-01 17:34:38,767, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-01 17:37:41,616, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 32
2023-02-01 17:37:41,632, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 59
2023-02-01 17:37:41,719, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-01 17:37:43,311, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-01 17:37:47,234, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-01 17:38:17,967, ERROR, Сбой в работе программы: 'list' object has no attribute 'get',Файл - homework.py, Строка № 119
2023-02-01 17:40:32,171, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 32
2023-02-01 17:40:32,177, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 59
2023-02-01 17:40:32,263, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-01 17:40:34,085, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-01 17:40:39,886, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-01 17:41:07,045, ERROR, Сбой в работе программы: list indices must be integers or slices, not str,Файл - homework.py, Строка № 119
2023-02-01 17:46:19,559, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 32
2023-02-01 17:46:19,565, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 59
2023-02-01 17:46:19,685, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-01 17:46:21,216, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-01 17:47:02,848, INFO, Старт проверки наличия токенов,Файл - homework.py, Строка № 32
2023-02-01 17:47:02,864, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 59
2023-02-01 17:47:02,964, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-01 17:47:04,561, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-01 17:47:08,635, INFO, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-01 17:47:20,837, INFO, Старт проверки статусов ДЗ,Файл - homework.py, Строка № 85
2023-02-01 17:47:37,681, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-02-01 17:47:38,805, DEBUG, {'chat': {'username': 'slexvik', 'type': 'private', 'id': 397784581, 'last_name': 'S', 'first_name': 'Alexey'}, 'supergroup_chat_created': False, 'delete_chat_photo': False, 'caption_entities': [], 'text': 'Изменился статус проверки работы "Slexvik__homework_bot.zip". Работа проверена: у ревьюера есть замечания.', 'message_id': 142, 'date': 1675248456, 'group_chat_created': False, 'new_chat_photo': [], 'entities': [], 'channel_chat_created': False, 'new_chat_members': [], 'photo': [], 'from': {'username': 'MyTrainerRobot', 'is_bot': True, 'first_name': 'My trainer robot', 'id': 5762476788}},Файл - bot.py, Строка № 131
2023-02-01 17:47:38,805, DEBUG, Exiting: send_message,Файл - bot.py, Строка № 132
2023-02-01 17:47:43,855, DEBUG, Сообщение отправлено: Изменился статус проверки работы "Slexvik__homework_bot.zip". Работа проверена: у ревьюера есть замечания.,Файл - homework.py, Строка № 54
2023-02-01 18:38:27,625, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-01 18:38:27,625, CRITICAL, Отсутствует обязательная переменная окружения:['TELEGRAM_CHAT_ID'] Программа принудительно остановлена,Файл - homework.py, Строка № 41
2023-02-01 18:56:42,781, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-01 18:56:42,800, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 60
2023-02-01 18:56:42,899, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-01 18:56:45,142, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-01 18:56:48,056, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 78
2023-02-01 18:57:00,448, ERROR, Сбой в работе программы: Ответ должен быть словарем, а не <class 'list'>,Файл - homework.py, Строка № 123
2023-02-01 19:02:14,037, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-01 19:02:14,043, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 60
2023-02-01 19:02:14,136, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-01 19:02:16,476, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-01 19:02:18,259, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 78
2023-02-01 19:02:29,297, ERROR, Сбой в работе программы: 'NoneType' object has no attribute '__name__',Файл - homework.py, Строка № 123
2023-02-01 19:03:14,193, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-01 19:03:14,200, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 60
2023-02-01 19:03:14,288, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-01 19:03:16,321, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-01 19:03:20,481, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 78
2023-02-01 19:03:25,486, ERROR, Сбой в работе программы: Ответ должен быть словарем, а не list,Файл - homework.py, Строка № 123
2023-02-01 19:07:43,696, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-01 19:07:43,705, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 60
2023-02-01 19:07:43,823, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-01 19:07:46,597, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-01 19:07:48,822, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 78
2023-02-01 19:08:02,844, ERROR, Сбой в работе программы: Homeworks должен быть списком, а не <class 'NoneType'>,Файл - homework.py, Строка № 124
2023-02-01 21:24:36,998, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-01 21:24:37,017, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 60
2023-02-01 21:24:37,111, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-01 21:24:39,317, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-01 21:24:49,862, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 78
2023-02-01 21:24:58,967, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-01 21:25:30,070, ERROR, Сбой в работе программы: list indices must be integers or slices, not str,Файл - homework.py, Строка № 124
2023-02-01 21:25:46,241, DEBUG, Старт отправки сообщения,Файл - homework.py, Строка № 48
2023-02-01 21:25:47,273, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-02-01 21:25:48,828, DEBUG, {'date': 1675261549, 'chat': {'first_name': 'Alexey', 'type': 'private', 'id': 397784581, 'username': 'slexvik', 'last_name': 'S'}, 'text': 'Сбой в работе программы: list indices must be integers or slices, not str', 'new_chat_photo': [], 'new_chat_members': [], 'supergroup_chat_created': False, 'caption_entities': [], 'message_id': 143, 'photo': [], 'channel_chat_created': False, 'delete_chat_photo': False, 'entities': [], 'group_chat_created': False, 'from': {'is_bot': True, 'first_name': 'My trainer robot', 'id': 5762476788, 'username': 'MyTrainerRobot'}},Файл - bot.py, Строка № 131
2023-02-01 21:25:48,843, DEBUG, Exiting: send_message,Файл - bot.py, Строка № 132
2023-02-01 21:26:03,611, DEBUG, Сообщение отправлено: Сбой в работе программы: list indices must be integers or slices, not str,Файл - homework.py, Строка № 55
2023-02-02 10:06:56,541, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-02 10:06:56,553, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 60
2023-02-02 10:06:56,650, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 10:06:58,384, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675307216 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-02-02 10:07:04,973, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 78
2023-02-02 10:07:08,500, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 10:07:20,548, INFO, 1675307218,Файл - homework.py, Строка № 122
2023-02-02 10:07:31,098, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 60
2023-02-02 10:07:31,165, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 10:07:32,612, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675307218 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-02-02 10:07:36,048, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 78
2023-02-02 10:07:38,126, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 10:07:43,950, INFO, 1675307252,Файл - homework.py, Строка № 122
2023-02-02 10:30:20,240, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-02 10:30:20,256, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 10:30:20,351, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 10:30:22,267, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675308620 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-02-02 10:30:25,810, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-02 10:30:36,888, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 10:30:43,283, INFO, 1675308622,Файл - homework.py, Строка № 122
2023-02-02 10:34:07,379, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-02 10:34:07,397, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 10:34:07,481, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 10:34:08,900, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675308847 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-02-02 10:34:08,936, ERROR, Сбой в работе программы: list index out of range,Файл - homework.py, Строка № 128
2023-02-02 10:34:08,938, DEBUG, Старт отправки сообщения,Файл - homework.py, Строка № 48
2023-02-02 10:34:08,941, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-02-02 10:34:11,039, DEBUG, {'new_chat_members': [], 'delete_chat_photo': False, 'new_chat_photo': [], 'text': 'Сбой в работе программы: list index out of range', 'message_id': 144, 'entities': [], 'photo': [], 'supergroup_chat_created': False, 'group_chat_created': False, 'chat': {'username': 'slexvik', 'last_name': 'S', 'id': 397784581, 'type': 'private', 'first_name': 'Alexey'}, 'caption_entities': [], 'date': 1675308850, 'channel_chat_created': False, 'from': {'first_name': 'My trainer robot', 'is_bot': True, 'username': 'MyTrainerRobot', 'id': 5762476788}},Файл - bot.py, Строка № 131
2023-02-02 10:34:11,039, DEBUG, Exiting: send_message,Файл - bot.py, Строка № 132
2023-02-02 10:34:11,051, DEBUG, Сообщение отправлено: Сбой в работе программы: list index out of range,Файл - homework.py, Строка № 53
2023-02-02 10:35:30,337, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-02 10:35:42,734, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 10:35:48,523, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 10:35:49,992, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675308930 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-02-02 10:36:14,784, ERROR, Сбой в работе программы: list index out of range,Файл - homework.py, Строка № 128
2023-02-02 10:37:14,921, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-02 10:37:19,268, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 10:37:23,498, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 10:37:24,981, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 10:37:31,938, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-02 10:37:43,929, ERROR, Сбой в работе программы: Пустой ответ от API,Файл - homework.py, Строка № 128
2023-02-02 10:58:43,900, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-02 10:58:43,907, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 10:58:44,002, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 10:58:45,714, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 10:58:45,775, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-02 11:03:16,104, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-02 11:03:16,114, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 11:03:16,195, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 11:03:17,600, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 11:03:17,631, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-02 11:03:25,918, INFO, В ответе от API нет ключа homeworks,Файл - homework.py, Строка № 127
2023-02-02 11:15:51,914, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-02 11:15:51,921, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 11:15:52,008, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 11:15:53,504, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 11:15:53,562, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-02 11:16:04,521, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 11:16:27,024, DEBUG, Старт проверки статусов ДЗ,Файл - homework.py, Строка № 90
2023-02-02 11:17:20,228, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-02 11:17:20,234, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 11:17:20,315, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 11:17:22,060, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 11:17:22,096, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-02 11:17:27,823, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 11:17:37,697, DEBUG, Старт проверки статусов ДЗ,Файл - homework.py, Строка № 90
2023-02-02 11:18:18,752, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 11:18:18,861, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 11:18:20,248, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 11:18:22,682, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-02 11:18:28,612, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 11:19:53,197, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-02 11:19:53,204, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 11:19:53,310, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 11:19:54,784, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 11:19:54,814, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-02 11:20:03,295, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 11:20:10,227, DEBUG, Старт проверки статусов ДЗ,Файл - homework.py, Строка № 90
2023-02-02 11:20:13,406, DEBUG, Изменился статус проверки работы "Slexvik__homework_bot.zip". Работа проверена: у ревьюера есть замечания.,Файл - homework.py, Строка № 117
2023-02-02 11:20:30,242, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 11:20:30,313, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 11:20:31,704, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 11:20:33,636, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-02 11:20:38,998, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 11:20:47,467, DEBUG, Старт проверки статусов ДЗ,Файл - homework.py, Строка № 90
2023-02-02 11:20:54,265, DEBUG, Изменился статус проверки работы "Slexvik__homework_bot.zip". Работа проверена: у ревьюера есть замечания.,Файл - homework.py, Строка № 117
2023-02-02 11:48:19,291, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-02 11:48:26,362, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 11:48:26,478, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 11:48:27,967, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 11:48:29,862, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-02 11:48:29,864, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 11:48:42,159, DEBUG, Старт проверки статусов ДЗ,Файл - homework.py, Строка № 90
2023-02-02 11:48:53,385, DEBUG, Старт отправки сообщения,Файл - homework.py, Строка № 48
2023-02-02 11:48:54,172, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-02-02 11:48:55,281, DEBUG, {'group_chat_created': False, 'chat': {'id': 397784581, 'type': 'private', 'username': 'slexvik', 'last_name': 'S', 'first_name': 'Alexey'}, 'date': 1675313335, 'caption_entities': [], 'text': 'Изменился статус проверки работы "Slexvik__homework_bot.zip". Работа проверена: у ревьюера есть замечания.', 'new_chat_photo': [], 'channel_chat_created': False, 'message_id': 145, 'supergroup_chat_created': False, 'photo': [], 'new_chat_members': [], 'entities': [], 'delete_chat_photo': False, 'from': {'id': 5762476788, 'is_bot': True, 'username': 'MyTrainerRobot', 'first_name': 'My trainer robot'}},Файл - bot.py, Строка № 131
2023-02-02 11:48:55,296, DEBUG, Exiting: send_message,Файл - bot.py, Строка № 132
2023-02-02 11:48:56,727, DEBUG, Сообщение отправлено: Изменился статус проверки работы "Slexvik__homework_bot.zip". Работа проверена: у ревьюера есть замечания.,Файл - homework.py, Строка № 53
2023-02-02 11:49:16,340, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 11:49:21,676, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 11:49:23,080, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 11:49:31,231, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-02 11:49:31,242, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 11:49:40,620, DEBUG, Статус ДЗ не изменился,Файл - homework.py, Строка № 120
2023-02-02 11:49:53,238, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 11:49:53,337, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 11:49:55,116, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 11:49:58,043, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-02 11:49:58,046, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 11:50:04,417, DEBUG, Статус ДЗ не изменился,Файл - homework.py, Строка № 120
2023-02-02 11:50:16,409, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 11:50:16,476, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 11:50:17,851, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 11:50:19,059, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 76
2023-02-02 11:50:19,063, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 11:50:23,671, DEBUG, Статус ДЗ не изменился,Файл - homework.py, Строка № 120
2023-02-02 11:50:57,244, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 34
2023-02-02 11:51:09,880, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 11:51:30,845, ERROR, Сбой в работе программы: local variable 'response' referenced before assignment,Файл - homework.py, Строка № 130
2023-02-02 11:51:37,116, DEBUG, Старт отправки сообщения,Файл - homework.py, Строка № 48
2023-02-02 11:51:39,383, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-02-02 11:51:40,565, DEBUG, {'message_id': 146, 'new_chat_members': [], 'chat': {'id': 397784581, 'last_name': 'S', 'username': 'slexvik', 'type': 'private', 'first_name': 'Alexey'}, 'supergroup_chat_created': False, 'text': "Сбой в работе программы: local variable 'response' referenced before assignment", 'new_chat_photo': [], 'entities': [], 'delete_chat_photo': False, 'group_chat_created': False, 'photo': [], 'caption_entities': [], 'date': 1675313500, 'channel_chat_created': False, 'from': {'username': 'MyTrainerRobot', 'is_bot': True, 'id': 5762476788, 'first_name': 'My trainer robot'}},Файл - bot.py, Строка № 131
2023-02-02 11:51:40,584, DEBUG, Exiting: send_message,Файл - bot.py, Строка № 132
2023-02-02 11:51:44,351, DEBUG, Сообщение отправлено: Сбой в работе программы: local variable 'response' referenced before assignment,Файл - homework.py, Строка № 53
2023-02-02 11:51:55,915, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 11:52:02,175, ERROR, Сбой в работе программы: local variable 'response' referenced before assignment,Файл - homework.py, Строка № 130
2023-02-02 11:52:18,445, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 58
2023-02-02 11:52:22,250, ERROR, Сбой в работе программы: local variable 'response' referenced before assignment,Файл - homework.py, Строка № 130
2023-02-02 12:00:34,362, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 43
2023-02-02 12:00:34,371, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 67
2023-02-02 12:00:34,454, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 12:00:36,352, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 12:00:39,619, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 12:00:52,788, ERROR, Сбой в работе программы: Ответ должен быть словарем, а не list,Файл - homework.py, Строка № 138
2023-02-02 12:02:37,432, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 43
2023-02-02 12:02:37,440, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 67
2023-02-02 12:02:37,532, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 12:02:39,027, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 12:02:45,233, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 12:02:57,159, ERROR, Сбой в работе программы: Ответ должен быть словарем, а не списком,Файл - homework.py, Строка № 138
2023-02-02 12:04:23,780, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 43
2023-02-02 12:04:23,799, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 67
2023-02-02 12:04:23,888, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 12:04:25,376, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 12:04:28,105, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 12:04:35,529, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 94
2023-02-02 12:04:42,910, ERROR, Сбой в работе программы: list indices must be integers or slices, not str,Файл - homework.py, Строка № 138
2023-02-02 12:06:35,891, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 43
2023-02-02 12:06:35,905, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 67
2023-02-02 12:06:35,995, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 12:06:37,515, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 12:07:00,419, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 43
2023-02-02 12:07:00,437, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 67
2023-02-02 12:07:00,522, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 12:07:01,988, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 12:07:32,922, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 85
2023-02-02 12:07:41,827, ERROR, Сбой в работе программы: Homeworks должен быть списком, а не списком ,Файл - homework.py, Строка № 138
2023-02-02 13:59:46,378, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 68
2023-02-02 13:59:46,463, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 13:59:48,553, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675321186 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-02-02 13:59:48,587, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 86
2023-02-02 13:59:48,589, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 95
2023-02-02 13:59:48,591, ERROR, Сбой в работе программы: list index out of range,Файл - homework.py, Строка № 137
2023-02-02 13:59:48,593, DEBUG, Старт отправки сообщения,Файл - homework.py, Строка № 57
2023-02-02 13:59:48,595, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-02-02 13:59:55,506, ERROR, Сообщения в телеграм не отправляются, Chat_id is empty,Файл - homework.py, Строка № 61
2023-02-02 14:01:33,720, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 43
2023-02-02 14:01:33,724, CRITICAL, Отсутствует обязательная переменная окружения:['TELEGRAM_CHAT_ID'] Программа принудительно остановлена,Файл - homework.py, Строка № 50
2023-02-02 14:02:00,830, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 69
2023-02-02 14:02:00,909, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 14:02:03,340, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675321320 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-02-02 14:02:03,387, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 87
2023-02-02 14:02:03,390, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 96
2023-02-02 14:02:03,393, ERROR, Сбой в работе программы: list index out of range,Файл - homework.py, Строка № 138
2023-02-02 14:02:03,395, DEBUG, Старт отправки сообщения,Файл - homework.py, Строка № 57
2023-02-02 14:02:03,399, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-02-02 14:02:09,371, ERROR, Сообщения в телеграм не отправляются, Chat_id is empty,Файл - homework.py, Строка № 61
2023-02-02 14:05:45,975, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 69
2023-02-02 14:05:49,926, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 14:05:51,792, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 14:06:01,098, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 87
2023-02-02 14:06:01,106, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 96
2023-02-02 14:06:05,558, DEBUG, Старт проверки статусов ДЗ,Файл - homework.py, Строка № 101
2023-02-02 14:06:09,004, DEBUG, Старт отправки сообщения,Файл - homework.py, Строка № 57
2023-02-02 14:06:09,715, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-02-02 14:06:14,988, ERROR, Сообщения в телеграм не отправляются, Chat_id is empty,Файл - homework.py, Строка № 61
2023-02-02 14:06:23,707, ERROR, Сообщения в телеграм не отправляются, Chat_id is empty,Файл - homework.py, Строка № 133
2023-02-02 14:06:56,146, DEBUG, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 69
2023-02-02 14:06:58,699, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 14:07:00,967, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675125074 HTTP/1.1" 200 943,Файл - connectionpool.py, Строка № 456
2023-02-02 14:07:03,840, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 87
2023-02-02 14:07:03,843, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 96
2023-02-02 14:07:06,696, DEBUG, Старт проверки статусов ДЗ,Файл - homework.py, Строка № 101
2023-02-02 14:07:08,930, DEBUG, Старт отправки сообщения,Файл - homework.py, Строка № 57
2023-02-02 14:07:09,625, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-02-02 14:07:12,295, ERROR, Сообщения в телеграм не отправляются, Chat_id is empty,Файл - homework.py, Строка № 61
2023-02-02 14:46:22,834, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 43
2023-02-02 14:46:22,834, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 68
2023-02-02 14:46:22,864, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 14:46:25,762, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675323982 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-02-02 14:46:25,771, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 86
2023-02-02 14:46:25,772, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 95
2023-02-02 14:46:25,773, ERROR, Сбой в работе программы: list index out of range,Файл - homework.py, Строка № 134
2023-02-02 14:46:25,774, DEBUG, Старт отправки сообщения,Файл - homework.py, Строка № 57
2023-02-02 14:46:25,776, DEBUG, Entering: send_message,Файл - bot.py, Строка № 129
2023-02-02 14:46:28,494, DEBUG, {'new_chat_members': [], 'caption_entities': [], 'chat': {'last_name': 'S', 'id': 397784581, 'first_name': 'Alexey', 'type': 'private', 'username': 'slexvik'}, 'photo': [], 'message_id': 147, 'group_chat_created': False, 'new_chat_photo': [], 'date': 1675323988, 'text': 'Сбой в работе программы: list index out of range', 'entities': [], 'delete_chat_photo': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 5762476788, 'first_name': 'My trainer robot', 'is_bot': True, 'username': 'MyTrainerRobot'}},Файл - bot.py, Строка № 131
2023-02-02 14:46:28,504, DEBUG, Exiting: send_message,Файл - bot.py, Строка № 132
2023-02-02 14:46:28,507, DEBUG, Сообщение отправлено: Сбой в работе программы: list index out of range,Файл - homework.py, Строка № 63
2023-02-02 14:56:28,513, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 68
2023-02-02 14:56:28,537, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 14:56:30,090, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675323982 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-02-02 14:56:30,099, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 86
2023-02-02 14:56:30,100, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 95
2023-02-02 14:56:30,101, ERROR, Сбой в работе программы: list index out of range,Файл - homework.py, Строка № 134
2023-02-02 14:56:36,227, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 43
2023-02-02 14:56:42,462, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 68
2023-02-02 14:56:48,945, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 14:56:51,003, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675324596 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-02-02 14:57:05,042, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 86
2023-02-02 14:58:32,764, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 95
2023-02-02 15:03:14,577, DEBUG, Старт проверки наличия токенов,Файл - homework.py, Строка № 43
2023-02-02 15:03:19,904, INFO, Старт запуская обращения к АПИ,Файл - homework.py, Строка № 68
2023-02-02 15:03:23,972, DEBUG, Starting new HTTPS connection (1): practicum.yandex.ru:443,Файл - connectionpool.py, Строка № 1003
2023-02-02 15:03:25,971, DEBUG, https://practicum.yandex.ru:443 "GET /api/user_api/homework_statuses/?from_date=1675324994 HTTP/1.1" 200 42,Файл - connectionpool.py, Строка № 456
2023-02-02 15:03:33,628, DEBUG, Старт проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 86
2023-02-02 15:03:37,376, DEBUG, Конец проверки АПИ на соответствие документацией,Файл - homework.py, Строка № 95
